src/modules/Dashboard/features/Applets/Table/AppletItem/AppletItem.tsx

```diff
@@ -49,9 +55,7 @@ export const AppletItem = ({ item }: { item: FolderApplet }) => {
   };
 
   const checkAppletEncryption = (callback: () => void) =>
-    item?.roles?.includes('owner') && !item?.encryption
-      ? setPasswordPopupVisible(true)
-      : callback();
+    item.role?.includes('owner') && !item.encryption ? setPasswordPopupVisible(true) : callback();
```

Comment:
may be it does make sense to make a constant for 'owner'? or may be to make an utility function to check for 'owner' role?